modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/RosettaError.scala

```diff
@@ -49,6 +49,13 @@ object RosettaError extends IntEnum[RosettaError] with RosettaErrorEncoder {
         retriable = false,
         description = Some("The operation is not supported.")
       )
+  case class InternalError(reason: String)
+      extends RosettaError(
+        value = 8,
+        message = "Internal error",
+        retriable = false,
```

Comment:
Why internal error is not retriable? IMO the internal state can change and it won't throw an internal server error anymore